app/components/ui/Cards/FormattedDate.tsx

```diff
@@ -0,0 +1,36 @@
+import { StrapiModel } from "models/Strapi";
+import React from "react";
+import formatEventDate from "utils/formatEventDate";
+import formatEventTime from "utils/formatEventTime";
+
+const ERROR_MESSAGE = "Invalid Date: missing property {startdate}";
+
+// Renders a human-friendly date
+export const FormattedDate = ({
+  calendarEvent,
+}: {
+  calendarEvent: StrapiModel.CalendarEvent;
+}): JSX.Element => {
+  if (calendarEvent?.startdate === undefined) {
```

Comment:
I think we want a more user friendly error message here. I suggest to check with Kyle about the message and/or if the start date should be required.

Reply:
You know what, lets make an executive decision and mark this field as _required_. There's no world where an event doesn't have a start date. 😅

Reply:
Done: https://github.com/Exygy/our415-strapi/pull/6